pkg/kms/webkms/remotekms.go

```diff
@@ -384,7 +394,52 @@ func (r *RemoteKMS) PubKeyBytesToHandle(pubKey []byte, kt kms.KeyType) (interfac
 //  - error if import failure (key empty, invalid, doesn't match KeyType, unsupported KeyType or storing key failed)
 func (r *RemoteKMS) ImportPrivateKey(privKey interface{}, kt kms.KeyType,
 	opts ...kms.PrivateKeyOpts) (string, interface{}, error) {
-	return "", nil, errors.New("function ImportPrivateKey is not implemented in remoteKMS")
+	destination := r.keystoreURL + "/import"
+
+	keyBytes, err := x509.MarshalPKCS8PrivateKey(privKey)
+	if err != nil {
+		return "", nil, fmt.Errorf("failed to marshal private key: %w", err)
+	}
```

Comment:
I would suggest adding the same switch kt as in the kms server here

Reply:
will target this in the follow-up ticket https://github.com/hyperledger/aries-framework-go/issues/2563